src/preset.js

```diff
@@ -44,4 +44,12 @@ module.exports = {
 
 	// Default is "node", but we need browser APIs
 	testEnvironment: 'jsdom',
+
+	// Load node exports (CommonJS)
```

Comment:
This isn't to load the CJS output, no. As mentioned, Jest (w/ jsdom) will consume Preact's browser build while (incorrectly) applying Node extension semantics -- i.e., `.js` in a non-`"type": "module"` project will be treated as CJS.

What we need to do is ensure Jest doesn't consume the browser build, despite this being a mistake on their end. Users can still get ESM, just through the `"import"` condition instead.

https://github.com/preactjs/preact/pull/3634